roles/remediate/tasks/leapp_multiple_kernels.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: leapp_multiple_kernels | Boot to latest kernel and remove old kernels
+  block:
+    - name: leapp_multiple_kernels | Get list of installed kernels packages sorted by the newest version
+      ansible.builtin.shell: |
+        set -o pipefail
+        sudo rpm -q kernel --queryformat '%{version}-%{release}.%{arch}\n' | sort -Vr
+      register: installed_kernels
+      changed_when: true
+
+    - name: leapp_multiple_kernels | Check current kernel version
+      ansible.builtin.command: uname -r
+      register: current_kernel
+      changed_when: true
+
+    - name: leapp_multiple_kernels | Set default kernel to latest
+      ansible.builtin.command: grubby --set-default /boot/vmlinuz-{{ installed_kernels.stdout_lines[0] }}
+      when: installed_kernels.stdout_lines[0] != current_kernel.stdout
+      register: set_default_kernel
+      changed_when: set_default_kernel.rc == 0
+
+    - name: leapp_multiple_kernels | Reboot into new kernel
+      ansible.builtin.shell: sleep 2 && shutdown -r now
+      async: 1
+      poll: 0
+      failed_when: false
+      when: installed_kernels.stdout_lines[0] != current_kernel.stdout
+      changed_when: true
+
+    - name: leapp_multiple_kernels | Wait for reboot
+      ansible.builtin.wait_for_connection:
+        delay: 15
+        timeout: 60
+      become: false
+      when: installed_kernels.stdout_lines[0] != current_kernel.stdout
```

Comment:
In other roles in this collection where a reboot is required, we are using the reboot module like this example:
```
- name: update-and-reboot | Reboot when updates applied
  ansible.builtin.reboot:
    reboot_timeout: 7200
    post_reboot_delay: "{{ post_reboot_delay }}"
```
It would be nice to do the same here to be consistent. At the very least, we should increase the timeout to `7200`.

Same comments apply to `leapp_newest_kernel_not_in_use.yml`.